packages/next/server/config.ts

```diff
@@ -23,6 +23,7 @@ import {
 } from '../shared/lib/image-config'
 import { loadEnvConfig } from '@next/env'
 import { hasNextSupport } from '../telemetry/ci-info'
+import semver from 'semver'
```

Comment:
```suggestion
import { gte as semverGte } from 'next/dist/compiled/semver'
```

Import the pre-compiled version of `semver` instead.